pkg/reconcile/pipeline/context/impl_test.go

```diff
@@ -233,6 +222,7 @@ var _ = Describe("Context", func() {
 
 			_, err := ctx.Applications()
 			Expect(err).To(HaveOccurred())
+			fmt.Println(err.Error())
```

Comment:
do we really need this line?

Reply:
removed 7c194b11d0a69dc090b73eed1edae3707e476d68